src/test/java/seedu/mtracker/console/AddCryptoParserTest.java

```diff
@@ -76,37 +107,67 @@ public int getParameterSize() {
     }
 
     @Test
-    void addCryptoParams_noRemarks_expectSuccess() {
+    void addCryptoParams_noRemarks_expectSuccess() throws OperationAbortedError {
         testCryptoParameters(USER_INPUT_NO_REMARKS, EXPECTED_PARAMS_NO_REMARKS);
     }
 
     @Test
-    void addCryptoParams_allValidParametersWithRemarksAndExpiry_expectSuccess() {
+    void addCryptoParams_allValidParametersWithRemarksAndExpiry_expectSuccess() throws OperationAbortedError {
         testCryptoParameters(USER_INPUT_WITH_REMARKS_AND_EXPIRY,
                 EXPECTED_PARAMS_WITH_REMARKS_AND_EXPIRY);
     }
 
     @Test
-    void addCryptoParams_tryInvalidNameMultipleTimes_expectSuccess() {
+    void addCryptoParams_tryInvalidNameMultipleTimes_expectSuccess() throws OperationAbortedError {
         testCryptoParameters(USER_INPUT_TRY_INVALID_NAME, EXPECTED_PARAMS_NO_REMARKS);
     }
 
     @Test
-    void addCryptoParams_tryInvalidPriceMultipleTimes_expectSuccess() {
+    void addCryptoParams_tryInvalidPriceMultipleTimes_expectSuccess() throws OperationAbortedError {
         testCryptoParameters(USER_INPUT_TRY_INVALID_PRICE,
                 EXPECTED_PARAMS_WITH_REMARKS_AND_EXPIRY);
     }
 
     @Test
-    void addCryptoParams_tryInvalidSentimentMultipleTimes_expectSuccess() {
+    void addCryptoParams_tryInvalidSentimentMultipleTimes_expectSuccess() throws OperationAbortedError {
         testCryptoParameters(USER_INPUT_TRY_INVALID_SENTIMENT,
                 EXPECTED_PARAMS_WITH_REMARKS_AND_EXPIRY);
     }
 
     @Test
-    void addCryptoParams_tryEmptyExpiryMultipleTimes_expectSuccess() {
+    void addCryptoParams_tryEmptyExpiryMultipleTimes_expectSuccess() throws OperationAbortedError {
         testCryptoParameters(USER_INPUT_TRY_EMPTY_EXPIRY,
                 EXPECTED_PARAMS_WITH_REMARKS_AND_EXPIRY);
     }
 
+    // @@KVignesh122
+    @Test
+    void addCryptoParams_abortAtName_expectException() {
+        assertThrows(OperationAbortedError.class,
+            () -> testCryptoParameters(USER_INPUT_TRY_ABORT_AT_NAME, EXPECTED_PARAMS_NO_REMARKS));
```

Comment:
```suggestion
            () -> testCryptoParameters(USER_INPUT_TRY_ABORT_AT_NAME, NO_PARAMS_EXPECTED));
```
Can create an empty string[], cause the params are redundant at that point